### PR TITLE
Fix build warnings

### PR DESCRIPTION
**Pull request by Narre**

### CODE REVIEW

File: src/gui/menu_manager.hpp [hunk 1]
```diff
@@ -46,9 +46,9 @@ class MenuManager final : public Currenton<MenuManager>
 private:
   template<typename T>
   struct SettableItem {
-    std::unique_ptr<T> current;
-    std::unique_ptr<T> next;
-    bool has_next;
+    std::unique_ptr<T> current = nullptr;
+    std::unique_ptr<T> next = nullptr;
+    bool has_next = false;
```
Comment (mrkubax10) [inline src/gui/menu_manager.hpp:51]: I am not sure about variable initalization in header files.

Comment (Vankata453) [reply src/gui/menu_manager.hpp:51]: Same.

Comment (Narre) [reply src/gui/menu_manager.hpp:51]: I tried everything I could think of, everything else either didn't work or at least didn't get rid of the warnings. Do you know how to do this properly?

Comment (mrkubax10) [reply src/gui/menu_manager.hpp:51]: I suppose that it should be fine like it is.

**Pull request merged**